composer/trainer/trainer.py

```diff
@@ -750,16 +763,18 @@ def __init__(
         self.adaptive_gradient_accumulation = _is_adaptive_grad_accum(grad_accum, device=self._device)
         grad_accum = _get_initial_grad_accum(grad_accum)
 
+        # Run Name
+        generated_run_name = _get_run_name(run_name=run_name)
```

Comment:
nit: since it could also be the user-supplied `run_name`, would prefer to keep the variable called `run_name`
```suggestion
        run_name = _get_run_name(run_name=run_name)
```